### PR TITLE
[emscripten] Use long instead of int for ssize_t

### PR DESCRIPTION
**Pull request by binji**
This was recently changed in emscripten.

### CODE REVIEW

File: src/config.h.in [hunk 1]
```diff
@@ -279,7 +279,7 @@ int wabt_vsnprintf(char* str, size_t size, const char* format, va_list ap);
 #endif
 
 #if !HAVE_SSIZE_T
-typedef int ssize_t;
+typedef long ssize_t;
 #endif
```
Comment (kripken) [inline src/config.h.in]: Hmm, is `ssize_t` not already defined by clang in emscripten?

Comment (kripken) [reply src/config.h.in]: Oh, it would be in a C header, I guess? I suppose this config.h runs before including it?

Comment (binji) [reply src/config.h.in]: I think I disabled the check for emscripten in cmake. 
See https://github.com/WebAssembly/wabt/blob/master/CMakeLists.txt#L64-L71

Not sure why I did it now -- perhaps it didn't work at the time. I'll see if I can remove it.

Comment (binji) [reply src/config.h.in]: Nice, looks like it works for emscripten, just breaks for msvc.

**Pull request merged**